sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectKey=DFE-Digital_npq-registration
```

Comment:
`DFE-Digital_early-careers-framework` ?

Reply:
whoops, good spot, it doesn't actually use this as we pass it in directly in the CI call but I've updated 👍 thanks